tests/e2e/device/e2e_device_commands.c

```diff
@@ -0,0 +1,1150 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */

```

Comment:
I'm assuming (not so naively I believe) that this is refactored implementation from Azure RTOS, which itself has got extensive review?

If I'm wrong let me know and especially if there's areas you think may need more review, otherwise just resolve this comment.

Reply:
Analogous feedback for the other test implementations of .c files.

Reply:
this is similar to azrtos device side code .